src/components/organisms/InformationLeftColumn/InformationLeftColumn.tsx

```diff
@@ -1,83 +1,101 @@
-import React from "react";
-import "./InformationLeftColumn.scss";
+import React, {
+  forwardRef,
+  useCallback,
+  useImperativeHandle,
+  useState,
+} from "react";
+import classNames from "classnames";
+
+import { IconDefinition } from "@fortawesome/fontawesome-common-types";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import {
   faAmbulance,
   faAngleDoubleRight,
   faHeart,
   faEdit,
+  faInfoCircle,
 } from "@fortawesome/free-solid-svg-icons";
 
-interface PropsType {
-  venueLogoPath: string;
+import "./InformationLeftColumn.scss";
+
+export type ValidLogoIconName = "ambulance" | "create" | "heart" | "info";
+
+const logoMap = new Map<ValidLogoIconName, IconDefinition>([
+  ["ambulance", faAmbulance],
+  ["create", faEdit],
+  ["heart", faHeart],
+  ["info", faInfoCircle],
+]);
+
+interface InformationLeftColumnProps {
+  iconNameOrPath?: ValidLogoIconName | string;
   children: React.ReactNode;
-  isLeftColumnExpanded: boolean;
-  setIsLeftColumnExpanded: React.Dispatch<React.SetStateAction<boolean>>;
 }
 
-const InformationLeftColumn: React.FunctionComponent<PropsType> = ({
-  venueLogoPath,
-  children,
-  isLeftColumnExpanded,
-  setIsLeftColumnExpanded,
-}) => {
+export interface InformationLeftColumnControls {
```

Comment:
export type

Reply:
```suggestion
export type InformationLeftColumnControls = {
```

Reply:
I believe my usage here is correct, as per the following:

- https://blog.logrocket.com/types-vs-interfaces-in-typescript/

> **Types vs. interfaces**
>
> The difference between types and interfaces in TypeScript used to be more clear, but with the latest versions of TypeScript, they’re becoming more similar.
> 
> **Interfaces** are basically a way to **describe data shapes, for example, an object.**
> 
> **Type** is a definition of a type of data, for example, **a union, primitive, intersection, tuple, or any other type.**

Reply:
Added this to the documentation PR in https://github.com/sparkletown/sparkle/pull/1062#issuecomment-772978074